src/services/FilterService.ts

```diff
@@ -0,0 +1,33 @@
+import { Filters } from '../types/Filters/Filters';
```

Comment:
this file name shouldn't be in PascalCase but in camelCase. Pascal case for files with React components

Also, I'd rename it - helpers/utils instead of service (optionally)